src/parsers/manifestjson.js

```diff
@@ -456,6 +456,12 @@ export default class ManifestJSONParser extends JSONParser {
       this.collector.addNotice(messages.MANIFEST_UNUSED_UPDATE);
     }
 
+    if (this.parsedJSON.granted_host_permissions) {
```

Comment:
This implementation doesn't warn when the field is set, but to false. The schema file results in a false value by default.


If that's desirable, consider adding a unit test that confirms that setting the value to false doesn't do anything.

Reply:
Yeah, I guess we don't need to warn for `granted_host_permissions: false`.

Reply:
I added a test case to cover the behavior expected when granted_host_permissions is set to false.